### PR TITLE
[Fix] fix mmcv-full to  mmcv==2.0.0rc2

### PR DESCRIPTION
**Pull request by vansin**
fix mmcv-full to  mmcv==2.0.0rc2

### CODE REVIEW

**Review by irexyc — APPROVED** [review]

File: docs/en/get_started.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ We recommend that users follow our best practices installing MMDeploy.
 
 ```shell
 pip install -U openmim
-mim install mmcv-full
+mim install "mmcv==2.0.0rc2"
```
Comment (RunningLeon) [inline docs/en/get_started.md:67]: pls do not lock version and change to `mmcv>=2.0.0rc2`

File: docs/zh_cn/get_started.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ conda install pytorch=={pytorch_version} torchvision=={torchvision_version} cpuo
 
 ```shell
 pip install -U openmim
-mim install mmcv-full
+mim install "mmcv==2.0.0rc2"
```
Comment (RunningLeon) [inline docs/zh_cn/get_started.md:63]: pls do not lock version and change to `mmcv>=2.0.0rc2`

File: tests/regression/mmdet.yml [hunk 3]
```diff
@@ -169,7 +169,7 @@ models:
   - name: YOLOV3
     metafile: configs/yolo/metafile.yml
     model_configs:
-      - configs/yolo/yolov3_d53_320_273e_coco.py
+      - configs/yolo/yolov3_d53_8xb8-320-273e_coco.py
```
Comment (RunningLeon) [inline tests/regression/mmdet.yml:172]: hasn't it changed in https://github.com/open-mmlab/mmdeploy/pull/1334?

Comment (vansin) [reply tests/regression/mmdet.yml:172]: don't know why GitHub detect the change

**Pull request closed without merging**